src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -499,6 +499,9 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
 
     DEBUG_LOG(PrintableModel(*nGraphFunc, "org_"));
 
+    Transformations transformations(nGraphFunc, enableLPT, inferencePrecision, isLegacyAPI(), snippetsMode, engConfig);
```

Comment:
So now we create transformation pipeline with engine config instead of merged engine + compilation config. This is ok since it doesn't affect transformations behavior, but in fact logically incorrect. We will need to refactor this in follow-up PRs. 
The idea is to merge engine and compilation configs via `readProperties` call and use the merged one everywhere during compilation. It will also allow to get rid of `getInferencePrecision`, `getSnippetsMode` and other similar functions.